doc/website/blog/2021-07-29-generate-a-fully-working-go-crud-http-api-with-ent.md

```diff
@@ -147,7 +139,7 @@ func (Pet) Fields() []ent.Field {
 ```
 
 We added two fields to our `Pet` entity: `name` and `age`. The `ent.Schema` just defines the fields of our entity. To
-generate runnable code from our schema, run:
+generate runnable code from our schema, run (and ignore the dependency issues for now):
```

Comment:
this isn't clear. what dependency issue?

Reply:
If you have a look [here](https://github.com/masseelch/elk/blob/41b7573b4b2e21bcb53bd27cb540e6a27778e18f/easyjson.go#L33) after the ent and elk templates have been generated the hook executes the easyjson generator. After the generation of ent / elk the new packages (like `masseelch/render` or `mailru/easyjson` are not yet reflected in the `go.mod`. This happens in the `go run -mod=mod step`. The easyjson generator however collects information about the go ast and this requires compilation and therefore the `go generate ./...` command reports missing dependencies in the `go.mod`/`go.sum`. 

To circumvent this the user has to run the generator without easyjson, run create a main function, `go get` the dependencies, run the generator again and then start the server. 

I am clueless, how to prevent this. If you have any idea I am very happy to try it out 😊

Reply:
I changed it back and added a third step when setting up the extension: Pull in dependecies manually.